Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -524,6 +524,21 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Returns a format localized string.
+    ///
+    ///    "%d Swift %d Objective-C".formatLocalized(1, 2) -> 1 Swift 2 Objective-C
+    ///
+    /// - Parameters:
+    ///   - comment: Optional comment for translators.
+    ///   - arguments: Arguments used by format.
+    /// - Returns: Format localized string.
+    func formatLocalized(comment: String = "", _ arguments: CVarArg...) -> String {
+        let format = localized(comment: comment)
```

Comment:
One of the requirements in this project is to not reference other functions so that any function can be copied individually.
```suggestion
        let format = NSLocalizedString(self, comment: comment)
```